src/middleware/sanitizeRequestBody.js

```diff
@@ -5,7 +5,8 @@ const emoji = require('node-emoji');
 const { flow } = require('lodash');
 
 const sanitizeRequestBody = (req, res, next) => {
-  const santizeValue = flow([emoji.strip, sanitizer.sanitize]);
+  // Strip emojis, remove scripts then restore special characters
+  const santizeValue = flow([emoji.strip, sanitizer.sanitize, sanitizer.unescapeEntities]);
```

Comment:
is there a method in `sanitizer` to sanitise without escaping entities? saves running two functions?

Reply:
Not sure, its is a pretty old module (https://www.npmjs.com/package/sanitizer, 4+ years). It doesn't seem like it does. We may need to change to an up to date module for customisation functionality.